2-mandatory/1-weather-report.js

```diff
@@ -12,8 +13,19 @@
 */
 
 function getTemperatureReport(cities) {
```

Comment:
This implementation looks good 👍 
As mentioned below, you may just need to check that you're using `let` or `const` when declaring a new variable. Even though the code works without it, it's a good habit to get into - otherwise we'll run into problems when we work with larger code bases 😄